src/ui/templates/insurance/your-business/company-details.njk

```diff
@@ -98,109 +99,97 @@
+          errorMessage: validationErrors.errorList.tradingName,
+          headerClass: 'govuk-fieldset__legend--m',
+          datacyHeading: 'trading-name-heading'
+        }) }}
+      </div>
+
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+          {{ yesNoRadioButtons.render({
+            fieldId: FIELDS.YOUR_COMPANY.TRADING_ADDRESS,
+            heading: CONTENT_STRINGS.TRADING_ADDRESS,
+            submittedAnswer: submittedValues[FIELDS.YOUR_COMPANY.TRADING_ADDRESS],
+            errorMessage: validationErrors.errorList.tradingAddress,
+            headerClass: 'govuk-fieldset__legend--m',
+            datacyHeading: 'trading-address-heading'
+          }) }}
+      </div>
+
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+        {{ govukInput({
+          label: {
+            text: CONTENT_STRINGS.WEBSITE,
+            classes: "govuk-label--m",
             isPageHeading: false,
-            classes: "govuk-fieldset__legend--m"
-          },
-          attributes: {
-            'data-cy': 'trading-name'
-          }
-        },
-        items: [
-          {
-            value: "yes",
-            text: "Yes",
-            attributes: {
-              'data-cy': "trading-name-yes"
-            }
-          },
-          {
-            value: "no",
-            text: "No",
             attributes: {
-              'data-cy': "trading-name-no"
-            }
+            'data-cy': 'company-website-heading'
           }
-        ]
-      }) }}
-    </div>
-
-    <div class="govuk-grid-column-three-quarters-from-desktop">
-      {{ govukRadios({
-        classes: "govuk-radios--inline",
-        name: "trading-address",
-        fieldset: {
-          legend: {
-            text: CONTENT_STRINGS.TRADING_ADDRESS,
-            isPageHeading: false,
-            classes: "govuk-fieldset__legend--m"
           },
+          id: FIELDS.YOUR_COMPANY.WEBSITE,
+          name: FIELDS.YOUR_COMPANY.WEBSITE,
+          classes: "input-width",
           attributes: {
-            'data-cy': 'trading-address'
+            'data-cy': 'company-website',
+            'aria-label': CONTENT_STRINGS.WEBSITE
           }
-        },
-        items: [
-          {
-            value: "yes",
-            text: "Yes",
+        }) }}
+      </div>
+
+      <div class="govuk-grid-column-three-quarters-from-desktop">
+        {{ govukInput({
+          label: {
+            text: CONTENT_STRINGS.PHONE_NUMBER,
+            classes: "govuk-label--m",
+            isPageHeading: false,
             attributes: {
-              'data-cy': "trading-address-yes"
+            'data-cy': 'company-phone-heading'
             }
           },
-          {
-            value: "no",
-            text: "No",
+          hint: {
+            text: CONTENT_STRINGS.PHONE_NUMBER_HINT,
             attributes: {
-              'data-cy': "trading-address-no"
+              'data-cy': 'company-phone-hint'
```

Comment:
Can we use the field ID for all data-cy selectors? So we don't have it hardcoded and then easier to update in the future eg

`'data-cy': FIELDS.YOUR_COMPANY.PHONE_NUMBER + '-hint'`

Reply:
I am doing this as I go along incase I change anything.  Can I leave this for now until I reach the ticket for this?